openapi/azure.openai.embeddings/Module.md

```diff
@@ -0,0 +1,67 @@
+## Overview
+This is a generated connector from [Azure OpenAI Embeddings API v2023-03-15-preview](https://learn.microsoft.com/en-us/azure/cognitive-services/openai/reference#embeddings/) OpenAPI specification.
```

Comment:
Do we need to mention the version? If so can't we mention the version in a different way?

Reply:
Its not mentioned in the azure timeseries and textanalytics connectors - so we can remove it I think

Reply:
Resolved through 140c11110e38029ee8094b18de8f3e70c028c340